package.json

```diff
@@ -38,12 +38,13 @@
         "vscode:prepublish": "npm run package",
         "compile": "webpack",
         "watch": "webpack --watch",
-        "package": "webpack --mode production --devtool hidden-source-map",
+        "package": "npm run publish-lsp-server && webpack --mode production --devtool hidden-source-map",
```

Comment:
When building a dotnet project, Actions continue to work when the build of the project reported errors